assets/src/containers/GradeDistribution.js

```diff
@@ -48,14 +48,6 @@ function GradeDistribution (props) {
                   <Grid item xs={12} lg={10}>
                     <Histogram
                       data={gradeData.map(x => x.current_grade)}
-                      tip={createToolTip(d => renderToString(
```

Comment:
@justin0022 I have removed the tool-tip from the bars as it is redundant. Sorry if I caused any confusion on this. All the info that D3 tool-tip is displaying is already present either in the side table or count of students on the bar itself

Reply:
Sounds good, not a problem!